shared/js/trackers.js

```diff
@@ -1,16 +1,13 @@
 
 // these are defined in abp.js
-var abp,
-    easylists,
-    whitelists,
-    surrogates,
-    trackerWhitelist = {}
```

Comment:
`easylists` and `whitelists` are now defined further below. `surrogates` is redefined a few lines under this comment, and `trackerWhitelist` wasn't used.